apis/deployer/healthchecks/doc.go

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// Package core is the internal version of the API.
+// +k8s:deepcopy-gen=package
+// +k8s:openapi-gen=true
+// +k8s:defaulter-gen=TypeMeta
+
+// +groupName=healthchecks.deployer.landscaper.gardener.cloud
```

Comment:
I think this is not needed as we do not have specific typed objects but these are rather helper structs

Reply:
Done